Description: Fixing Makefile installation path
 The original upstream makefile installs the program to /usr/bin, but we
 want it under /usr/games
 .
 We also need to create our own installation directory, since the makefile
 assumes it already exists.
Author: Felix Crux <felixc@felixcrux.com>Index: b/Makefile
===================================================================
--- a/Makefile
+++ b/Makefile
@@ -4,13 +4,14 @@
 CFLAGS=-O2 -Wall -Wextra
 
 PROG=gti
-PREFIX=$(DESTDIR)/usr/bin
+PREFIX=$(DESTDIR)/usr/games
 
 $(PROG): *.c
 	$(CC) -o $@ $(CFLAGS) $^
 	-strip -s $@
 
 install: $(PROG)
+	mkdir -p $(PREFIX)
 	cp $^ $(PREFIX)
 
 uninstall:
